torch/csrc/jit/passes/quantization.cpp

```diff
@@ -611,23 +614,21 @@ class InsertQuantDeQuantHelper {
   void run(
     script::Module& module,
     const std::string& method_name);
-  // quantization parameters and scalar type
-  std::tuple<IValue, IValue> getQParams(script::Module& module, Value* v);
+  // quantization parameters including scale, zero_point,
+  // scalar_type and axis(for per channel quantization)
+  QParamMap getQParamMap(script::Module& module, Value* v);
   c10::optional<script::Module> findChildModuleToQuantize(
       script::Module& module,
       Value* child_instance);
   void collectObserverNodesAndValueToQuantize(script::Module& module, Value*);
   void removeObservers(script::Module& module, Graph* g);
-  bool registerQParams(script::Module& module,
-                       const std::tuple<IValue, IValue>& qparams_and_scalar_type,
-                       const std::string& prefix);
   void quantizeTensors(script::Module& module, Graph* g, Value* self);
 
  private:
   // TODO: we don't need to call this for each graph
```

Comment:
What does this TODO mean? Is it still relevant?

Reply:
yeah it's still relevant. Right now we store the key for `observer_modules_to_remove_` is graph, but actually it's per-module not per-graph, so ideally we should  store a map from `ModulePtr` to observer name.